### ISSUE TITLE
Fix how doc string looks in the documentation

### ISSUE THREAD

**Issue by MOchiara** [OceanGlidersCommunity/glidertest#156]
Original author part looks funcky and not nicely as parameters and returns
![image](https://github.com/user-attachments/assets/eb3a2594-731e-47be-997a-1868f471679c)

**Comment by callumrollo** [comment]
Not sure how easy that's gonna be. Parameters and Returns are established standard terms which is why they get the pretty formatting. You'd likely need to dig pretty deep in the sphinx docs engine to get it to treat Original author the same way. I think it looks fine as is tbh!

**Comment by MOchiara** [comment]
Yeah that is what we thought with Eleanor actually! I can have a look and if we do not fix it, it is also fine as it is

**Comment by callumrollo** [comment]
We're going to standardise on how compute_sunrise_sunset looks right now, with the addition of the author and notes as Eleanor suggestws
